packages/backend/server/src/core/workspaces/types.ts

```diff
@@ -52,22 +69,25 @@ export class InviteUserType extends OmitType(
 }
 
 @ObjectType()
-export class WorkspaceType implements Partial<Workspace> {
+export class WorkspaceFeatureType {
   @Field(() => ID)
   id!: string;
 
   @Field({ description: 'is Public workspace' })
   public!: boolean;
 
+  @Field({ description: 'Workspace created date' })
+  createdAt!: Date;
+}
+
+@ObjectType()
+export class WorkspaceType extends WorkspaceFeatureType {
   @Field({ description: 'Enable AI' })
   enableAi!: boolean;
 
   @Field({ description: 'Enable url previous when sharing' })
   enableUrlPreview!: boolean;
 
-  @Field({ description: 'Workspace created date' })
```

Comment:
Why

Reply:
moved into `WorkspaceFeatureType` and `WorkspaceType` extends it